tests/ssg_test_suite/common.py

```diff
@@ -245,6 +245,8 @@ def _make_file_root_owned(tarinfo):
     if tarinfo:
         tarinfo.uid = 0
         tarinfo.gid = 0
+        # set permission to 775
```

Comment:
Maybe this is too permissive? Would that be a problem?

Reply:
It shouldn't be a problem